low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -47,10 +48,12 @@ def __init__(self):
         self.server_frus = {
             'cpu': self.get_cpu_info,
             'platform_sensors': self.get_platform_sensors_info,
-            'memory': self.get_mem_info
+            'memory': self.get_mem_info,
+            'nw_ports': self.get_nw_ports_info,
         }
         self._ipmi = IpmiFactory().get_implementor("ipmitool")
         self.platform_sensor_list = ['Temperature', 'Voltage', 'Current']
+        self.nw_instance = NetworkInterface()
```

Comment:
Move this object creation under get_nw_status().
So that the object creation only happen when nw interface health info request come.

Reply:
I can move it under `get_nw_ports_info()` instead of `get_nw_status()` since it is getting called for each interface, so we will be creating the object multiple times.

Reply:
done 👍. Moved to get_nw_ports_info()